### DISCUSSION TITLE
macOS没办法成功安装sakuraFrp，请教一下各位大佬如何解决

### DISCUSSION THREAD

**Discussion by lujacob** [natfrp/wiki#408] [通常话题]
如图所示，安装时会出现以下情况，我的系统版本是10.15.7 ，是我电脑版本太旧了么
![Uploading 截屏2023-12-17 下午11.53.04.png…]()

[a comment by an automated account is not shown]

**Comment by fakeboboliu** [comment]
因为我们的苹果 GUI 使用了苹果在 2020 年之后发布的 swiftui 版本，苹果决定让 2020 年及之后的 swiftui 版本不支持 2019 年发布的系统，因此我们无法为您提供兼容，抱歉。

目前，您可以参考 Linux 教程自行启动 service 或 frpc。
